linting/stylelint-config-bitcrowd/index.js

```diff
@@ -1,4 +1,200 @@
 /* eslint-disable global-require */
-Object.assign = Object.assign || require('object-assign');
-
-module.exports = { extends: 'stylelint-config-standard', ...require('./overrides') };
+module.exports = {
+  extends: [
+    'stylelint-config-standard-scss',
+    'stylelint-config-prettier',
+  ],
+  plugins: [
+    'stylelint-order',
+  ],
+  rules: {
+    'order/properties-order': [
```

Comment:
💥 kinda excessive list 👀 

I know we had it before, but isn't this a bit of a pain to maintain?

Reply:
Actually this hasn’t been edited for a while — there are probably some missing CSS properties by now! — so the maintenance could be said to be minimal :D 

The reason for it is to keep the CSS readable — all the “positioning”/box-related stuff is in one place, all the font-related stuff etc. No need to hunt through the properties for other declarations that may be affecting layout. Similarly, we found it useful on tailwind projects to sort the order of classnames (the tailwind equivalent of sorting property order :D)

What if we switched to using an ordering package maintained by someone else? e.g. https://github.com/cahamilton/stylelint-config-property-sort-order-smacss seems like a sensible order, and is well-maintained

Reply:
Yep, I remember this ordering-disucssion from the [code guide](https://codeguide.co/#css-declaration-order) and I also think it makes perfect sense to have this.

Was also looking at other packages yesterday already and was surprised that they mostly do what we do: maintain a gigantic list.

So if you think we're able to maintain that, perfectly fine, otherwise using a list that someone else maintains and pulling that in would probably be easier on the long run. It would just be more people caring about the list to be up to date.

I had a look at exactly this package you were linking too. Can judge the efforts of switching to that and whether it "makes sense". Up to you with more knowledge on this particular repo here.

Reply:
Yes indeed, I think there’s no other way to handle this — either just alphabetise, or maintain a big list by hand 🙈 I’ll look into ordering with the smacss package I linked above, it would be good to not be responsible for this 👍